.github/CODEOWNERS

```diff
@@ -4,21 +4,22 @@
 # The last matching pattern takes precedence.
 # https://help.github.com/articles/about-codeowners/
 
-*                                           @backstage/reviewers
-/docs/features/techdocs                     @backstage/techdocs-core
-/docs/features/search                       @backstage/techdocs-core
-/docs/assets/search                         @backstage/techdocs-core
-/plugins/code-coverage                      @alde @nissayeva
-/plugins/code-coverage-backend              @alde @nissayeva
-/plugins/cost-insights                      @backstage/silver-lining
-/plugins/cloudbuild                         @trivago/ebarrios
-/plugins/search                             @backstage/techdocs-core
-/plugins/search-*                           @backstage/techdocs-core
-/plugins/techdocs                           @backstage/techdocs-core
-/plugins/techdocs-backend                   @backstage/techdocs-core
-/plugins/ilert                              @yacut
-/packages/search-common                     @backstage/techdocs-core
-/packages/techdocs-common                   @backstage/techdocs-core
-/.changeset/cost-insights-*                 @backstage/silver-lining
-/.changeset/search-*                        @backstage/techdocs-core
-/.changeset/techdocs-*                      @backstage/techdocs-core
+*                              @backstage/maintainers
```

Comment:
Shouldn't change this 😬

Reply:
Yes, did not intend for that to happen, no clue where it came from. Fixed in #06a255e